action.yml

```diff
@@ -9,10 +9,26 @@ inputs:
   all:
     description: "Whether to install every Nextflow release via the '-all' distribution."
     required: false
-    default: false
+    default: "false"
+  java-version:
+    description: "The Java version to install as the required dependency of Nextflow. Passed to the underlying setup-java action."
+    required: false
+    default: "17"
+  java-distribution:
+    description: "The Java distribution to install as the required dependency of Nextflow. Passed to the underlying setup-java action."
+    required: false
+    default: "zulu"
 runs:
-  using: "node20"
-  main: "dist/index.js"
+  using: "composite"
+  steps:
+    - uses: actions/setup-java@8df1039502a15bceb9433410b1a100fbe190c53b
+      with:
+        java-version: ${{ inputs.java-version }}
+        distribution: ${{ inputs.java-distribution }}
+    - uses: ./subaction
```

Comment:
this needs a checkout action before, otherwise it throws an error:
https://github.com/nf-core/methylseq/actions/runs/12174070795/job/33955445554?pr=478#step:3:123